transactions/Starly/acceptSaleOffer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "arguments": [
+        "itemID",
+        "marketCollectionAddress"
+    ],
+    "messages": {
+        "en": "Approve transaction if you want to make purchase"
```

Comment:
The message only needs to describe what the transaction does.
And you can use values from arguments.

For example it can be `Purchase StarlyCard ID#{itemID}`